orchestra/tests/test_dashboard.py

```diff
@@ -470,11 +472,12 @@ def test_assignment_snapshot_upgrades(self):
         good_snapshots = {
             'snapshots': [{
                 'data': {},
-                'datetime': datetime.utcnow().isoformat(),
+                'datetime': (
+                    datetime.utcnow().replace(tzinfo=tzutc()).isoformat()),
```

Comment:
At the very least `datetime.utcnow().replace(tzinfo=tzutc()).isoformat()` => `django.util.timezone.now().isoformat()`.

The django utils are nice wrappers that allow us not to think about timezones, I'm for using them whenever we can.